File: PreferenceSample/app/src/main/java/com/example/iyemon018/preferencesample/pref/DataSyncPrefModel.java
package com.example.iyemon018.preferencesample.pref;

import com.os.operando.garum.annotations.Pref;
import com.os.operando.garum.annotations.PrefKey;
import com.os.operando.garum.models.PrefModel;

@Pref
public final class DataSyncPrefModel extends PrefModel {
    
    @PrefKey("sync_frequency")
    public String syncFrequency;
    
}

File: PreferenceSample/app/src/main/java/com/example/iyemon018/preferencesample/DataSyncPrefModel.kt
package com.example.iyemon018.preferencesample

import com.os.operando.garum.annotations.Pref
import com.os.operando.garum.annotations.PrefKey
import com.os.operando.garum.models.PrefModel

@Pref
class DataSyncPrefModel : PrefModel() {

    @PrefKey("sync_frequency")
    public var syncFrequency: String = ""
}